src/aat/resources/features/F-037/S-022.td.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "must return negative response when request does not provide valid authentication credentials",
+	"_guid_": "S-022",
+	"_extends_": "F-037_Test_Data_Base",
+
+	"specs": [
+		"an active profile in CCD",
+		"does not provide valid authentication credentials",
+		"has the 403 return code"
+	],
+  "request": {
+    "headers": {
+      "Authorization": "eyJhbGciOiJIUzI1NiJ9"
```

Comment:
is this a valid Authorization or Invalid one?

Reply:
It is invalid one.